service/README.md

```diff
@@ -0,0 +1,33 @@
+# Service lifecycle
+
+Service, being a bit of an overloaded term, requires some
+specific care to explain the various components in the `service`
+package in UberFx.
+
+## Instantiation
+
+Generally, you create a service in one of two ways:
+
+* The builder pattern, e.g. `service.WithModules(...).Build()`
+* Calling `service.New()` directly.
+
+The former is generally much easier, and used in all the examples, but `New` is exported
+in case you'd like extra control over how your service is instantiated.
+
+If you **do** call `service.New()`, you will need to call `AddModules(...)` to configure
+which modules you'd like to serve.
+
+## Options
+
+Both the builder pattern and the `New()` functions take a variadic `Options`
+pattern, allowing you to pick and choose which components you'd like to
+override. As with many of the goals of UberFx, specify zero options should give
+you a fully working application.
+
+Once you have a service, you generally want to call `.Start()` on it.
```

Comment:
Any time you say something's generally a good idea, provide a reason, even if it seems obvious. If your audience needs to be told to call `.Start()`, they also probably need to be reminded of why.

Could you add a few words at the end of the sentence, like "to make sure that it runs" or whatever the reason is?

Also, could we just tell them to call `.Start()` instead of saying "generally want to"? If they don't want to, they can disobey.